pkg/bindings/kube/kube.go

```diff
@@ -94,3 +94,34 @@ func DownWithBody(ctx context.Context, body io.Reader) (*entities.KubePlayReport
 	}
 	return &report, nil
 }
+
+func DownKube(ctx context.Context, path string) (*entities.KubePlayReport, error) {
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		if err := f.Close(); err != nil {
+			logrus.Warn(err)
+		}
+	}()
+
+	return DownKubeWithBody(ctx, f)
+}
+
+func DownKubeWithBody(ctx context.Context, body io.Reader) (*entities.KubePlayReport, error) {
```

Comment:
Why the duplicates?

Reply:
I thought that the CI checks were failing because there was no /kube/down endpoint and the duplicates were my attempt at adding this. This was not why the CI checks were failing and the dupes will be removed in the next push.